### DISCUSSION TITLE
Min Browser and Keepass?

### DISCUSSION THREAD

**Discussion by ghost** [minbrowser/min#2088] [Q&A] [answered]
Hi!

#### Idea
I would be happy if I could import and export site passwords I access in Min browser in keepass format

#### question/feedback
What do you all think of the idea?

**Comment by barddust** [comment, marked as answer]
Also, Keepassxc

**Reply by ghost** [reply]
Hi fingerknight, thank you for feedback... but 
- Keepass and Keepassxc, are they the same format? 
- or Is there a difference between the two formats?

**Reply by barddust** [reply]
Actually, no idea. According to https://superuser.com/a/879013, *KeePassX is an "Contributed/Unofficial KeePassPort" of KeePass... KeePassXC seems to have been forked from KeePassX*.

In a word, KeePassXC derives from KeePass? (Maybe)

I have installed Keepass once before, but have never use it. Personally, KeePaxXC is much pretty than KeePass

**Reply by ghost** [reply]
Hi fingerknight, thank you for feedback again.
